tests/test_cameras/test_camera_parameter.py

```diff
@@ -5,7 +5,6 @@
 import pytest
```

Comment:
file name should be `test_camera_parameters.py`

Reply:
solved